build-system/tasks/helpers.js

```diff
@@ -717,14 +723,82 @@ function buildExaminer(options) {
  * @param {!Object} options
  */
 function buildWebWorker(options) {
-  const opts = Object.assign({}, options);
   return compileJs('./src/web-worker/', 'web-worker.js', './dist/', {
     toName: 'ww.max.js',
     minifiedName: 'ww.js',
     includePolyfills: true,
-    watch: opts.watch,
-    minify: opts.minify || argv.minify,
+    watch: options.watch,
+    minify: options.minify || argv.minify,
+  });
+}
+
+/**
+ * Build AMP experiments.js.
+ *
+ * @param {!Object} options
+ */
+function buildExperiments(options) {
+  return compileJs(
+    './build/experiments/',
+    'experiments.max.js',
+    './dist.tools/experiments/',
+    {
+      watch: false,
+      minify: options.minify || argv.minify,
+      includePolyfills: true,
+      minifiedName: 'experiments.js',
+    }
+  );
+}
+
+/**
+ * Build amp-login-done-${version}.js file.
+ *
+ * @param {string} version
+ * @param {!Object} options
+ */
+function buildLoginDone(version, options) {
```

Comment:
Move this back to `dist.js`, since it's only used in there.

Reply:
for consistency i feel like these should be here. (all the compileJs calls)

Reply:
The reason I suggested this was because `helpers.js` has bloated into a pretty large file, and we should try to separate functionality across the various compilation tasks. Also, there are `compileJs` calls in `build.js`, `dist.js`, `extension-helpers.js`, and other files too.

Fair enough though, let's punt this to a future refactor.

Reply:
its either that or we should move `buildAlp`, `buildExaminer` and the like from helpers and back to `dist.js` too

Reply:
The rationale I used was that if a function is used in more than one place, keep it in `helpers.js`. If it's used in one place, keep it in that file.
https://github.com/ampproject/amphtml/blob/7d98f1803eb5fc1370c99498e0998f0246bf2c86/build-system/tasks/build.js#L65-L72
https://github.com/ampproject/amphtml/blob/aec1991f4c3728c6a2461675024bdf547a4df124/build-system/tasks/dist.js#L91-L104

Reply:
done.